sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -44,7 +45,8 @@ import org.apache.spark.sql.util.CaseInsensitiveStringMap
 import org.apache.spark.storage.StorageLevel
 import org.apache.spark.unsafe.types.UTF8String
 
-class DataSourceV2Strategy(session: SparkSession) extends Strategy with PredicateHelper {
+class DataSourceV2Strategy(session: SparkSession) extends Strategy
+    with PredicateHelper with SQLConfHelper {
```

Comment:
We don't need `SQLConfHelper`. We can get the conf by `session.sessionState.conf`

Reply:
fixed.